publish_frame: odom
robot_frame: base_footprint
twist_topic: /vel
fix_topic: /fix
imu_topic: /imu
num_particles: 3000
publish_rate: 100
ess_threshold: 1000